scripts/exrrfs_ioda_bufr.sh

```diff
@@ -103,11 +120,14 @@ ${cpreq} "${USHrrfs}"/offline_domain_check_satrad.py .
 ${cpreq} "${USHrrfs}"/offline_ioda_tweak.py .
 for ioda_file in ioda*nc; do
   grid_file="${FIXrrfs}/meshes/${MESH_NAME}.static.nc"
+  #if [[ "${ioda_file}" == *abi* || "${ioda_file}" == *atms* ]]; then
```

Comment:
Do we want to keep the comment on line 123?

Reply:
Yes, I want to keep it for using the offline domain check for atms in the future.